rawdesign/resources/css/style.css

```diff
@@ -14,6 +14,23 @@ body {
     font-family: 'Mulish', sans-serif;
 }
 
+::-webkit-scrollbar {
+    width: 0.5rem;
+  }
+::-webkit-scrollbar-track {
+    background: repeating-linear-gradient(rgb(255, 87, 87), rgb(88, 88, 230), rgb(76, 228, 76),rgb(241, 241, 88));
+  }
+::-webkit-scrollbar-thumb {
+    background:#484849;
```

Comment:
Would have been better if the colors for the light theme were kept in the light.css file, works anyway 😅